deploy/templates/gcs-manifests/gcs-csi-virtblock.yml.j2

```diff
@@ -0,0 +1,247 @@
+## Deploy Gluster Virtual Block CSI attacher
+---
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: csi-attacher-glustervirtblockplugin
+  namespace: {{ gcs_namespace }}
+  labels:
+    app.kubernetes.io/part-of: gcs
+    app.kubernetes.io/component: csi-driver
+    app.kubernetes.io/name: csi-attacher
+spec:
+  serviceName: csi-attacher
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/part-of: gcs
+      app.kubernetes.io/component: csi-driver
+      app.kubernetes.io/name: csi-attacher
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/part-of: gcs
+        app.kubernetes.io/component: csi-driver
+        app.kubernetes.io/name: csi-attacher
+    spec:
+      serviceAccountName: csi-attacher
+      containers:
+        - name: csi-attacher
+          image: quay.io/k8scsi/csi-attacher:v1.0.1
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: MY_NAME
+              valueFrom:
+               fieldRef:
+                 fieldPath: metadata.name
+            - name: MY_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+
+        - name: glusterfs
+          image: docker.io/gluster/glustervirtblock-csi-driver:latest
+          args:
+            - "--nodeid=$(NODE_ID)"
+            - "--v=5"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--resturl=$(REST_URL)"
+            - "--resttimeout=120" #glusterd2 client timeout in sec
+          env:
+            - name: NODE_ID
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: CSI_ENDPOINT
+              value: unix://plugin/csi.sock
+            - name: REST_URL
+              value: http://glusterd2-client.{{ gcs_namespace }}:24007
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /plugin
+      volumes:
+        - name: socket-dir
+          emptyDir:
+
+## Deploy Gluster Virtual Block CSI NodePlugin
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: csi-nodeplugin-glustervirtblockplugin
```

Comment:
name can be ` csi-glustervirtblock-nodeplugin`

can this be done for others also

Reply:
Done